src/pages/billing/Project.js

```diff
@@ -276,12 +276,23 @@ const ProjectDetail = ({ project, project: { projectName }, billingAccounts, aut
 
   useEffect(() => StateHistory.update({ projectUsers }), [projectUsers])
 
-  // Render
-  const { displayName = null } = _.find({ accountName: billingAccountName }, billingAccounts) || { displayName: 'No Access' }
+  // There's some madness going on when using state variables in polling effects - the reference
+  // never updates (i.e. it's bound to the value that the component was first mounted with).
+  // `useGetter` works around this and I have no idea why.
```

Comment:
@zarsky-broad I'd appreciate your input here.

Reply:
not sure, but is `Utils.usePollingEffect` compatible with using updated state values? I see it uses `useOnMount`